### PR TITLE
Update to emit Kotlin Wire sources

### PR DESCRIPTION
**Pull request by Synesso**
Worked with @swankjesse to emit Kotlin Wire sources.

### CODE REVIEW

File: misk-grpc-tests/build.gradle [hunk 1]
```diff
@@ -28,9 +29,15 @@ protobuf {
   }
 }
 
+wire {
+  kotlin {
+  }
+}
```
Comment (mightyguava) [inline misk-grpc-tests/build.gradle]: Does this make the java code stop getting generated? Or does it just add the kotlin version?

Comment (Synesso) [reply misk-grpc-tests/build.gradle]: This merely adds the Kotlin sources. Tests are changed to ensure we're using those and not the Java ones.

**Comment by Synesso** [comment]
Waiting on release of wire `3.0.0-alpha02`, which fixes issues in alpha01 - as advised by @jrodbx

File: build.gradle [hunk 2]
```diff
@@ -1,3 +1,14 @@
+buildscript {
+  repositories {
+    maven {
+      url "https://maven.global.square/artifactory/square-public"
```
Comment (jjestrel) [inline build.gradle]: this shouldn't be here

File: misk-grpc-tests/build.gradle (hunk 1 above)
Comment (swankjesse) [reply misk-grpc-tests/build.gradle]: And we deleted the Java code!

Comment (swankjesse) [reply misk-grpc-tests/build.gradle]: Delete this too?

misk-grpc-tests/build-wire.sh

File: build.gradle (hunk 2 above)
Comment (jrodbx) [reply build.gradle]: correct, either swap in `mavenCentral()` here:
```
repositories {
  mavenCentral()
}
```

or add to the `plugins` block below:

```
plugins {
  ...
  id 'com.squareup.wire' version "3.0.0-alpha01"
}
```

File: misk-grpc-tests/build.gradle [hunk 3]
```diff
@@ -28,9 +29,15 @@ protobuf {
...
+
 sourceSets {
   main.java.srcDirs += 'build/generated/source/proto/main/grpc'
   main.java.srcDirs += 'build/generated/source/proto/main/java'
+  main.java.srcDirs += 'build/generated/src/main/java'
```
Comment (jrodbx) [inline misk-grpc-tests/build.gradle]: was this necessary?  the plugin should add this automatically.

File: misk-grpc-tests/src/test/kotlin/misk/grpc/MiskClientProtocServerTest.kt [hunk 4]
```diff
@@ -29,16 +29,10 @@ class MiskClientProtocServerTest {
...
-        .location(Point.Builder()
-            .latitude(43)
-            .longitude(-80)
-            .build())
-        .build())
+    val feature = grpcClient.call(grpcMethod, Point(latitude = 43, longitude = -80))
+    assertThat(feature).isEqualTo(Feature(
+        name = "pine tree",
+        location = Point(latitude = 43, longitude = -80)
+    ))
```
Comment (jrodbx) [inline misk-grpc-tests/src/test/kotlin/misk/grpc/MiskClientProtocServerTest.kt]: nice

File: misk-grpc-tests/build.gradle (hunk 3 above)
Comment (Synesso) [reply misk-grpc-tests/build.gradle]: Removed.

**Comment by swankjesse** [comment]
Let's merge now to unblock? Works as is.

**Comment by Egorand** [comment]
Sweet!

**Pull request merged**